spec/package_spec.rb

```diff
@@ -13,6 +13,74 @@
     end
   end
 
+  describe 'has_custom_object_requirements?' do
+    it 'should return true when custom object types are present' do
+      requirements_hash = {
+        'custom_objects' => {
+          'custom_object_types' => [
+            {
+              'key' => 'product',
+              'schema' => { 'properties' => { 'id' => { 'type' => 'string' } } }
+            }
+          ]
+        }
+      }
+      expect(described_class.has_custom_object_requirements?(requirements_hash)).to be(true)
+    end
+
+    it 'should return true when custom object relationships are present' do
+      requirements_hash = {
+        'custom_objects' => {
+          'custom_object_relationship_types' => [
+            { 'key' => 'a', 'source' => 'b', 'target' => 'a' }
+          ]
+        }
+      }
+
+      expect(described_class.has_custom_object_requirements?(requirements_hash)).to be(true)
+    end
+
+    it 'should return false when custom object types are not present' do
+      requirements_hash = {
+        'custom_objects' => {
+          'custom_object_types' => []
+        }
+      }
+
+      expect(described_class.has_custom_object_requirements?(requirements_hash)).to be(false)
+    end
+
+    it 'should return false when custom object relationships are not present' do
+      requirements_hash = {
+        'custom_objects' => {
+          'custom_object_relationship_types' => []
+        }
+      }
+
+      expect(described_class.has_custom_object_requirements?(requirements_hash)).to be(false)
+    end
+
```

Comment:
Edge case perhaps worth testing against: `should return true when custom object types are present but relationships types are not defined`, e.g.
```
      requirements_hash = {
        'custom_objects' => {
          'custom_object_types' => [
            {
              'key' => 'product',
              'schema' => { 'properties' => { 'id' => { 'type' => 'string' } } }
            }
          ],
		  'custom_object_relationship_types' => []
        }
```

Also the opposite, i.e. `should return true when custom object relationship types are present but custom object types are not defined`

Reply:
Done 👍